site/app/views/course/CourseMaterialsView.php

```diff
@@ -26,7 +26,7 @@ public function listCourseMaterials(array $course_materials_db) {
         $folder_ids = [];
         $links = [];
         $base_view_url = $this->core->buildCourseUrl(['course_material']);
-
+        $begining_of_time_date = "1900-01-01 00:00:00";
```

Comment:
Instead of hardcoding it here, you could add it to our [date utilities class](https://github.com/Submitty/Submitty/blob/main/site/app/libraries/DateUtils.php).